bundles/org.openhab.binding.wemo/README.md

```diff
@@ -51,7 +51,7 @@ This is combined with a delta trigger value, which is defaulted to 1 W.
 This means that the channel is only updated when one of the following conditions are met:
 
 1. The rounded value received is equal to the rounded average for the past minute, i.e. this value has stabilized. This introduces a delay for very small changes in consumption, but on the other hand it prevents excessive logging and persistence caused by temporary small changes and rounding.
-2. The rounded value received is more than 1 W from the previous value. So when changes are happening fast, the channel will also be updated fast.
+1. The rounded value received is more than 1 W from the previous value. So when changes are happening fast, the channel will also be updated fast.
```

Comment:
This makes numbering inconsitent, we now have 1. two timmes.

Reply:
^
In markdown, if you have several lines starting with '1.' it will auto-magically render sequentially as 1. / 2. / 3. etc.

Reply:
What @andrewfg says @hmerk

Vuepress (and i think even the github preview) will auto number those lists in a correct order.